### ISSUE TITLE
Prevent inserting formulas in the spreadsheet

### ISSUE THREAD

**Issue by OffCorner** [jamiewilson/form-to-google-sheets#3]
this little snippet is a true piece of art!
I have found a small issue though: it's possible to insert formulas into the spreadsheet. This could end up being a little annoying and given the fact that google Sheets is so powerful, it could lead to some vulnerabilities.
Anyways, in order to prevent formulas, I have added an apostrophe (') in front of every cell except for the timestamp using the following function:
`if(header !== 'timestamp') {`
`  return "'"+e.parameter[header]`
`}`
inserted on line 26 (or 68 in the commented version).

**Comment by g4spow** [comment]
Yes, good spot. This could be a problem. Maybe add in some validation preventing = in any of the fields

**Comment by radames** [comment]
also in addition to the apostrophe you could replace the = sign from the string
```js
e.parameter[header].replace(/=/g, '')
```
